misk/web/tabs/loader/src/components/ScriptComponent.tsx

```diff
@@ -1,22 +0,0 @@
-import { IWebTab } from "@misk/common"
-import * as React from "react"
```

Comment:
How come we don't need this file anymore?

Reply:
Got folded into `<TabLoaderComponent tabs={adminDashboardTabs} />`